examples/extensions/remove_entire_sitelink_campaign_extension_setting.py

```diff
@@ -0,0 +1,230 @@
+        print(
+            f'Request with ID "{ex.request_id}" failed with status '
+            f'"{ex.error.code().name}" and includes the following errors:'
+        )
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f"\t\tOn field: {field_path_element.field_name}")
+        sys.exit(1)
+
+
+def _create_sitelink_campaign_extension_setting_mutate_operation(
+    client, customer_id, campaign_id
+):
+    """Creates a MutateOperation for the sitelink campaign extension setting
+    that will be removed.
+
+    Args:
+        client: an initialized GoogleAdsClient instance
+        customer_id: the client customer ID.
+        campaign_id: the campaign ID.
+
+    Returns:
+        The created MutateOperation for the sitelink campaign extension
+            setting.
+    """
+    extension_type_enum = client.get_type("ExtensionTypeEnum", version="v6")
+    composite_id = ResourceName.format_composite(
+        campaign_id,
+        extension_type_enum.ExtensionType.Name(
+            extension_type_enum.ExtensionType.SITELINK
+        ),
+    )
+    # Construct the campaign extension setting resource name, in format:
+    # customers/{customer_id}/campaignExtensionSettings/{campaign_id}~{extension_type}
+    resource_name = client.get_service(
+        "CampaignExtensionSettingService", version="v6"
+    ).campaign_extension_setting_path(customer_id, composite_id)
+
+    # Create a MutateOperation for the campaign extension setting.
+    mutate_operation = client.get_type("MutateOperation", version="v6")
+    mutate_operation.campaign_extension_setting_operation.remove = resource_name
+    return mutate_operation
+
+
+def _get_all_sitelink_extension_feed_items(
+    client, ga_service, customer_id, campaign_id
+):
+    """Gets all sitelink extension feed items associated to the specified
+    campaign extension setting.
+
+    Args:
+        client: an initialized GoogleAdsClient instance.
+        ga_service: the Google Ads API service client.
+        customer_id: the client customer ID.
+        campaign_id: the ID of the campaign to get the sitelink extension feed
+            items from.
+
+    Returns:
+        An array of str resource names of extension feed items.
+    """
+    campaign_resource_name = client.get_service(
+        "CampaignService", version="v6"
+    ).campaign_path(customer_id, campaign_id)
+    extension_type_enum = client.get_type("ExtensionTypeEnum", version="v6")
+    extension_type_name = extension_type_enum.ExtensionType.Name(
+        extension_type_enum.ExtensionType.SITELINK
+    )
+
+    # Construct the query.
+    query = f"""
+        SELECT
+            campaign_extension_setting.campaign,
+            campaign_extension_setting.extension_type,
+            campaign_extension_setting.extension_feed_items
```

Comment:
These should only be indented 2 spaces not 4, since we use SQL style for these strings.